## Raising the Connectome: the emergence of neuronal activity and behavior in _C. elegans_

### Abstract
The differentiation of neurons and formation of connections between cells is the basis of both the adult phenotype and behaviors tied to cognition, perception, reproduction, and survival. Such behaviors are associated with local (circuits) and global (connectome) brain networks. A solid understanding of how these networks emerge is critical. Here, we provide a guided tour of early developmental events in their electrophysiological context. We demonstrate the transition from developmental cells to terminally differentiated cells as being defined by functional changes that set up adult behavioral function. In a model organism such as C. elegans, this transition is developmentally conserved and well-characterized in various ways by publicly-available data. In this paper, we will build on an existing time-series model of neuronal differentiation and conduct a functional assessment of this developmental connectome. Multiple sources of data will be brought to bear on this effort, providing information about the emerging connectome as well as the emergence of functional subnetworks. A first-mover model of the emerging connectome is also introduced, which will help to synthesize the details of our results. The broader implications of these functional analyses will be considered, including implications for computational modeling and behavioral origins.

### Introduction
The field of connectomics has provided opportunities to view the structure and function of nervous systems in a new light. The proliferation of data for small connectomes such as those found in the nematode Caenorhabditis elegans allows us to study the connected nervous system at both the single cell and systems level (Berck et.al, 2016; Eichler et.al, 2017; Ko et.al, 2013). In addition, emerging functional perspectives on genomic contributions (Barabasi and Barabasi, 2020) and statistical regularities (Takagi, 2017) of the connectome allow for a flexible, multiscale model of biology and behavior. In this paper, this perspective will be used to better understand the potential functional role of ion channels among terminally-differentiated neurons present at various points in embryogenesis. Based on these partial patterns of adult connectivity, generalizations can be made with respect to the structural and functional features of a developing connectome (Kaiser, 2017). These relationships tend to be conserved over developmental time, and changes due to postembryonic changes and larval plasticity are minimal. 

A parallel area of inquiry will be defined by the differentiation of developmental cells and the acquisition of their terminally-differentiated neuronal states. Far from being a switch operating at a discrete point in time, differentiation of developmental cells into neurons involves two sets of transcription factors: so-called terminal selectors (Hobert, 2016a) and protectors/stabilizers (Zheng and Chalfie, 2016). In the case of the former, transcriptional mechanisms serve to simultaneously initiate and maintain cell fate choice. Protector/stabilizers then ensure the smooth, uninterrupted expression of terminal selectors. The notion that neuronal differentiation is a complex process that unfolds over time is most important to the context of emerging connectomes. Therefore, it is imperative that details regarding the so-called neuronal identity of these newly-integrating cells be understood in order to predict the emergence of physiology and ultimately behavior (Hobert, 2016b). 

A third area of inquiry involves exactly when and under what conditions these terminally-differentiated neurons begin to produce organismal-level behaviors. It has been suggested that structural features emerge earlier in development than functional features (Cao et.al, 2016). As we will see, the first neuronal cells emerge well after the first synaptic connections between cells are established, which in turn is decoupled from the emergence of the first behaviors. Our contribution to this understanding is to flesh out the structure of this timing relative to differentiation events. This includes making connections between the terminal differentiation of specific neurons, their electrical and molecular milieu, and the systems-level significance of the temporal process. Hopefully, this brings us closer to understanding the tempo and scope of morphogenesis relative to the adult phenotype.   
One way to unify these sets of questions is by making reasonable assumptions about development from data collected in adult animals. Caenorhabditis elegans is the perfect choice for this approach due to its straightforward developmental trajectory. While there has been limited work done on the emergence of electrophysiology and electrical connectivity during the course of C. elegans embryogenesis, we can use a combination of data science and mathematically-based conceptual modeling to shed light on these relationships. An abundance of quantitative data also allows for inference to proceed using the lineage tree and adult end state in tandem to simulate intermediate developmental phenotypes.

In this paper, we provide a guided tour of early developmental events in electrophysiological context, along with a systems-level perspective on the electrophysiological milieu of the C. elegans connectome. Such a perspective is lacking in the C. elegans literature, and it should be broadly applicable to the study of developmental dynamics. We will build on the work done in Alicea (2018), where the construction of partial connectomes for multiple time points in embryonic development demonstrates how the neuronal networks emerge from developmental cells. We will also consider emerging neuronal networks as constituents of functional networks observed in the adult phenotype. Using information from multiple sources, we are able to fill in details along a timeline that spans from the birth of the first neuron to the emergence of the first behaviors. To make the connection between molecular biology, cell biology, and behavior, analogies can be made using both adult neuronal function and simulation methods.  

#### Justification of the _C. elegans_ model
This endeavor focuses on the _C. elegans_ connectome, and there are a number of reasons why limiting the analysis to this connectome is more illuminating than a broad cross-species study. The first of these is the deterministic nature of _C. elegans_ neurodevelopment. _C. elegans_ offers a clear mapping between developmental cell lineages, differentiating neurons in the embryo, and the adult phenotype. Secondly, this connectome is small but also functionally diverse, consisting of 302 neurons in the hermaphrodite (White et.al, 1986) and 385 cells in the male (Cook et.al, 2019). Related to this, all cells in the C. elegans connectome have a known function. A lack of functional ambiguity also provides us with both well-established patterns of connection and well-defined structural features (Varshney et.al, 2011). Finally, there is a well-established community of open resources that allow for multifaceted bioinformatic investigations.  
The _C. elegans_ model also allows us to look at developmental cell lineages (Sulston et.al, 1983) along with events that precede the emergence of the first neuronal cells. Figure 1 shows a complex distribution for the birth of developmental cells across the so-called first proliferation phase (Altun and Hall, 2011). While the birth of developmental cells occurs in aperiodic bursts during this interval, there are three bursts of particularly interest to our sampling of neuronal cell births. The first burst of births are shown in red, and occur from 210 to 285 minutes. Next is a burst of developmental cell births shown in orange, and occurs from 285 to 345 minutes. While there are no developmental cell births from 345 to 365 minutes, there is a third burst of births from 365-400 minutes of embryogenesis. Based on a prior analysis of the data (Alicea et.al, 2019), bursts of developmental cell births may precede rounds of neurogenesis. 

Even though _C. elegans_ has a deterministic developmental trajectory with respect to cell lineage and identity, this potential relationship is not straightforward. _C. elegans_ neurons undergo a multipotency-to-commitment transition (MCT), in which neuronal cells are born from a diverse developmental cell lineage that also gives rise to muscle and epithelial cells (Rothman and Jarriault, 2019). This may provide a basis for great electrophysiological diversity across cells in very distinct subcircuits relevant to behavior. As we will see, there is a difference between functional asymmetry and cell lineage asymmetry which is partially driven by the expression and function of ion channels.

<p align="center">
  <img width="256" height="227" src=""><BR>
  Figure 1. Developmental precursors to the first connectome cells to terminally-differentiate in the embryo. Data are arranged as a histogram with bins of size 8. Red, Orange, and Gray bars are bins of particular interest to our sampling scheme.
</p>
  
The aggregate measure shown in Figure 1 is consistent with the finding that cellular specialization is coordinated with rounds of cell division, although this does not result from a specific cell division-related mechanism (Alberts et.al, 2002). In conceptually separating the functional networks from structural networks, we find that different neuronal properties can provide alternate views of the developing connectome. For example, transcription factors define C. elegans neurons more distinctly than the 118 neuron classes defined by 118 neurons classes based on anatomical and synaptic connectivity (Hobert, 2016). Nevertheless, we do see that the first few neurons and major expansion in the number of neurons occur immediately after the first two bursts of developmental cell proliferation.

#### Ion Channels in _C. elegans_
We take the perspective that ion channels are critical to the embryonic, larval, and adult stages of life-history. In terms of the C. elegans nervous system, the larval stage exhibits the greatest plasticity of all life-history stages. In this paper, however, we are limiting our survey to ion channel expression in terminally-differentiated neurons. While little is known about functional ion channel expression in developmental cells, our goal is to identify the ion channels expressed in terminally-differentiated neurons on the way to adult behavioral function. While there are select studies on how membrane current is regulated in specific neuronal cell types in postembryonic development (for an example from AIY interneurons, see Faumont et.al, 2006), there are no system-wide surveys on the topic. Variation in ion channel expression can occur in three ways: synaptic pruning during sexual differentiation (Oren-Suissa et.al, 2016), defined mutant phenotypes (Jorgenson and Mango, 2002), and generalized phenotypic plasticity (Jin and Qi, 2018). In terms of phenotypic plasticity, an adaptive response generally occurs in postembryonic stages of development such as L1 (DD1 remodeling, see White, 1978) or Dauer (Procko et.al, 2011). More generally, we can observe adaptive plasticity throughout larval development and adulthood in specific functional circuits, such as the mechanosensory circuit (Bozorgmehr et.al, 2013). 

#### Outline of Approach
This analysis in this paper will involve four basic components. The first of these involves an analysis of cells at different points in time (Alicea et.al, 2018), allowing for an inventory of the various tissue and organ types represented by newly differentiated neurons. Ion channel data collected from WormBase (Harris et.al, 2019) will be used to identify key genes, associated cells, and annotations. Determining the relationships between functional gene identities, specific ion channels, and functional annotations will be enhanced by using a data set developed at the OpenWorm Foundation (Sarma et.al, 2018). This analysis yields information regarding ion channel types present at a specific time point in embryogenesis. In terms of the latter, the data set featured in (Packer et.al, 2019) can be used in conjunction with annotation metadata and embryonic gene expression information to draw conclusions regarding the phylogenetic and developmental origins of the connectome. Of particular interest is the link between ion channels expressed by neuronal cells present in the embryo and the lineage-specific expression of ion channel-related and other associated genes. The final component relies upon how stereotypical C. elegans behaviors are heavily influenced by genetics to establish a link between the presence of certain neurons and the potential for early forms of chemosensation, thermosensation, and even learning and memory (Walker et.al, 2019).

### Methods
__ChannelWorm.__ An annotated collection of ion channel functions by terminally-differentiated cell was used as an initial query of which cells are associated with which ion channels. The ChannelWorm dataset.also includes functional annotations that are both ion channel and cell specific.

__DevoWorm resources.__ The DevoWorm group has assembled a number of bioinformatic conversions that have been used in (Alicea, 2018) to track the birth times and developmental lineage of each cell in the C. elegans connectome. These data can be found on Github (https://github.com/devoworm/DevoWorm) and the Open Science Framework (doi:10.17605/OSF.IO/NG833).

__First-mover Model repository.__ All notes, figures, and code associated with the first-mover model are located in a Github repository (https://github.com/devoworm/Raising-the-Worm- Brain).

__VisCello.__ To discover further relationships between gene expression and ion channel expression in individual cells, we utilized the VisCello database (Packer et.al, 2019) which can be explored using a standalone app downloadable from Github (https://github.com/qinzhu/VisCello.celegans) or interactively via Shiny app (https://cello.shinyapps.io/celegans_explorer/). Plots produced using VisCello are produced using the UMAP (Uniform Manifold Approximation and Projection) method (https://umap-learn.readthedocs.io/en/latest/how_umap_works.html), which maps data points to a two-dimensional manifold (McInnes et.al, 2018). A form of dimensionality reduction that relies upon Topological Data Analysis, UMAP preserves global structure in the data, and uses graphical separation between related points to describe this structure.

__WormBase.__ Supplemental information about protein expression and cells of interest are retrieved from WormBase Version:WS273 (https://www.wormbase.org/). WormBase (Harris et.al, 2019) is a C. elegans community resource that allows us to map between specific adult cells.

### Results
This analysis will proceed by walking through the developmental connectome as it exists at five points during embryogenesis: 265, 280, 290, 300, and 400 minutes post-fertilization. These networks are defined in Alicea (2018), and are derived from information regarding the birth time of terminally-differentiation neurons rather than direct observations of connectivity. In addition to cells that are connected by common time of birth, subnetworks defined in the literature in terms of physiological function (particularly the NSY-5 subnetwork) will be analyzed. Figure 2 contains a timeline that shows major events of embryonic development in context. 

